app/src/main/java/com/halim/adam/cashmaster/DatabaseHelper.java

```diff
@@ -34,55 +60,642 @@ public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {
     }
 
     /*
-    INSERT TO TABLES ---------------------------------------------------------------------------------------------
+    INSERT TO TABLES
      */
 
-    /**
-     * Insert to table 'category'
-     * @param name name of the category
-     */
     public void InsertCategory(String name){
         SQLiteDatabase db = this.getWritableDatabase();
         String sql = "INSERT INTO category (name) VALUES ('" + name + "');";
```

Comment:
It would be better to use prepared statement because it's much more secure than appending the variable directly to the SQL string

https://developer.android.com/reference/java/sql/PreparedStatement